brainfuck.html.markdown

```diff
@@ -8,6 +8,8 @@ contributors:
 Brainfuck (not capitalized except at the start of a sentence) is an extremely
 minimal Turing-complete programming language with just 8 commands.
 
+You can try brainfuck on your browser with <a href="http://fatiherikli.github.io/brainfuck-visualizer/">brainfuck-visualizer</a>.
```

Comment:
This is written in HTML and not Markdown. You mind converting it into Markdown please? 

You can try brainfuck on your browser with <a href="http://fatiherikli.github.io/brainfuck-visualizer/">brainfuck-visualizer</a>.

to 

You can try brainfuck on your browser with [brainfuck-visualizer](http://fatiherikli.github.io/brainfuck-visualizer).